packages/@o3r/core/schematics/rule-factories/otter-environment/index.ts

```diff
@@ -18,15 +18,23 @@ import {
   TYPES_DEFAULT_FOLDER,
 } from '@o3r/schematics';
 import generateEnvironments from '@schematics/angular/environments/index';
+import type {
+  TsConfigJson,
+} from 'type-fest';
 import * as ts from 'typescript';
 
 const editTsConfigJson = (tree: Tree) => {
-  if (tree.exists('/tsconfig.json')) {
-    const tsConfig = ts.parseConfigFileTextToJson('/tsconfig.json', tree.readText('/tsconfig.json')).config;
-    if (tsConfig.compilerOptions?.noPropertyAccessFromIndexSignature) {
-      delete tsConfig.compilerOptions.noPropertyAccessFromIndexSignature;
+  const tsConfigPath = '/tsconfig.json';
```

Comment:
should not be instead something like:

```suggestion
  const tsConfigPath =['/tsconfig.lib.jdon', '/tsconfig.app.json', '/tsconfig.json'].filter((file) => tree.exists(file));
```

and run on all the file found?

Reply:
this is only about the root `tsconfig` file so there won't be app/lib ones

Reply:
waiting for https://github.com/AmadeusITGroup/otter/pull/2648 to be merged and to adapt this PR